src/main/resources/swagger.api/schemas/models.yaml

```diff
@@ -13,9 +13,23 @@ MosaicOrder:
       type: boolean
       default: true
 
+MosaicOrderRequest:
+  type: object
+  properties:
+    placeholder:
```

Comment:
since we have contract defined now, we do not need placeholder any more.
Also MosaicOrder model defined in this file is not used now

Reply:
it is still need to avoid api-lint issue because our dto inside of acq-models file